sanket/javascript/assignment14(day26)/sanketA14_Q1.html

```diff
@@ -0,0 +1,41 @@
+<html>
+  <head>
+    <title>Inheritance</title>
+  </head>
+  <body>
+    <script language="javascript">
+      class student {
+        constructor(RollNo, Name) {
+          this.RollNo = RollNo;
+          this.Name = Name;
+        }
+        display() {
+          document.write("<br>Inside Display");
+          document.write("<br>Roll No " + this.RollNo);
+          document.write("<br>Name " + this.Name);
+        }
+      }
+      //Marks is a derived class/child class
+      class Marks extends student {
+        //Constructor Overriding
+        constructor(RollNo, Name, Marks) {
+          super();
+          this.RollNo = RollNo;
+          this.Name = Name;
```

Comment:
this.RollNo = RollNo;
          this.Name = Name;
No need to initialize RollNo and Name as they will be initialized in the Super Class constructor.

Reply:
ho so can you please explain

Reply:
Remove this.Name and this.RollNo in Marks Class Because it's already define in Student class

Reply:
In Super method pass RollNo and Name as parameter.

Reply:
yes , sanket please make the changes and also declare d1 as date object like d1 = new Date();

Reply:
ma'am, i already created date object on line no 35 do you want me to create another one